scripts/exrrfs_mpassit.sh

```diff
@@ -34,6 +34,22 @@ YYYYMMDDHH=${CDATE}
 YYYYMMDD=${CDATE:0:8}
 HH=${CDATE:8:2}
 #
+# find init.nc
+#
+n=0
+while [[ $n -le 12 ]]; do
+  CDATEic=$($NDATE -$((10#${n})) ${CDATE})
+  YYYYMMDDic=${CDATEic:0:8}
+  HHic=${CDATEic:8:2}
+  checkfile=${COMINrrfs}/${RUN}.${YYYYMMDDic}/${HHic}/ic/${WGF}${MEMDIR}/init.nc
+  if [[ -s ${checkfile} ]]; then
+     ln -sf ${checkfile} .
```

Comment:
45,46: The Shellcheck Linter will require double quotes for `${checkfile}` to avoid any potential globbing or broken words.

Reply:
What do you mean by double quotes for ${checkfile}?

Reply:
"${checkfile}"